scripts/test/hyriseConsole_test.py

```diff
@@ -129,11 +129,15 @@ def main():
     console.sendline("print nation")
     console.expect("=== Chunk 3 ===")
 
-    # Clear generated tables.
+    # Clear all generated tables and restore default configuration.
     console.sendline("reset")
     console.sendline("select * from meta_tables")
     console.expect("0 rows total")
 
+    # Make sure to turn off binary table caching again.
```

Comment:
Why?

Reply:
It's there to ensure that no binaries are written/read at the same time because we call the console tests in parallel from the project root for clang-release and clang-debug/gcc-debug.

Reply:
Got it.